daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/ledgerinteraction/JsonLedgerClient.scala

```diff
@@ -439,6 +439,7 @@ class JsonLedgerClient(
         // TODO (MK) Using a grpc exception here doesn’t make that much sense.
         // We should refactor this to provide something more general.
         Future.successful(
+          // TODO error codes: Adapt V2 ?
```

Comment:
Language or runtime team?

Reply:
>    Language or runtime team?

Language is responsible from this part of the script. @stefanobaghino-da you can ignore this one.

Reply:
cc @cocreature

Reply:
that’s fine, just keep it